test/unit/ngsi20/examples/contextRequests/updateContext2.json

```diff
@@ -0,0 +1,10 @@
+{
```

Comment:
updateContext1.json and updateContext2.json seems to be the same file... Are both needed?

Reply:
You are right. I have just deleted updateContext2.json